spec/support/shared/shopify_request.rb

```diff
@@ -14,18 +14,17 @@
     ShopifyAPI::Base.user = nil
   end
 
-  def mock_request(endpoint, extension)
+  def mock_request(file, endpoint, extension)
```

Comment:
Transactions is part of the endpoint. It will reach all the transactions associated with the order, which we required in order to successfully create a refundable order.

I don't understand your `array or not` argument.